README.md

```diff
@@ -1,31 +1,95 @@
-
-## Installation
-
-SDP is officially supported for Python 3.10, but might work for other versions.
-
-To install all required dependencies run `pip install -r requirements/main.txt`. You will need to install
-additional requirements if you want to [run tests](tests/README.md) or [build documentation](docs/README.md).
-
-Some SDP processors depend on the NeMo toolkit (ASR, NLP parts) and NeMo Text Processing.
-Please follow [NeMo installation instructions](https://github.com/NVIDIA/NeMo#installation)
-and [NeMo Text Processing installation instructions](https://github.com/NVIDIA/NeMo-text-processing#installation)
-if you need to use such processors.
-
-## Contributing
-We welcome community contributions! Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) for the process.
+# Speech Data Processor (SDP) Toolkit
+
+The Speech Data Processor (SDP) is a toolkit designed to simplify the processing of speech datasets. It minimizes the boilerplate code required and allows for easy sharing of processing steps. SDP's philosophy is to represent processing operations as 'processor' classes, which take in a path to a NeMo-style data manifest as input (or a path to the raw data directory if you do not have a NeMo-style manifest to start with), apply some processing to it, and then save the output manifest file.
+
+## Features
+
+- **Creating Manifests:** Generate manifests for your datasets.
+- **Running ASR Inference:** Automatically run ASR inference to remove utterances where the reference text differs greatly from ASR predictions.
+- **Text Transformations:** Apply text-based transformations to lines in the manifest.
+- **Removing Inaccurate Transcripts:** Remove lines from the manifest which may contain inaccurate transcripts.
+- **Custom Processors:** Write your own processor classes if the provided ones do not meet your needs.
+
+## Installation
+
+SDP is officially supported for Python 3.10, but might work for other versions.
+
+1. Clone the repository:
+
+```bash
+   git clone https://github.com/NVIDIA/NeMo-speech-data-processor.git
+   cd NeMo-speech-data-processor
+```
+2. Install dependencies:
+```bash
+   pip install -r requirements/main.txt
+```
+
+3. Optional: If you need to use ASR, NLP parts, or NeMo Text Processing, follow the NeMo installation instructions:
+   - [NeMo Installation](https://github.com/NVIDIA/NeMo.git)
+
+## Example:
+1. In this example we will load librispeech using SDP.
+   * For downloading all available data - replace config.yaml with all.yaml
+   * For mini dataset - replace with mini.yaml.
+3. If you need something specific split or the combination of splits if one file
+   * Pass to optional argument data_split ["dev_clean", dev_test] (for example)
+```bash
+    python NeMo-speech-data-processor/main.py \
+    --config-path="dataset_configs/english/librispeech" \
+    --config-name="config.yaml" \
+    processors_to_run="0:" \
+    workspace_dir=" /app/librispeecht"

```

Comment:
Let's make this `workspace_dir="librispeech_data_dir"` or something, so that the script will be able to run if the user just copy-pastes the bash command?

Reply:
Yes, great idea!

Reply:
Done